pkg/server/home_workspaces.go

```diff
@@ -114,7 +117,7 @@ func WithHomeWorkspaces(
 		homePrefix:           homePrefix,
 		bucketLevels:         bucketLevels,
 		bucketSize:           bucketSize,
-		kcp:                  buildExternalClientsAccess(kubeClusterClient, kcpClusterClient),
+		kcp:                  buildExternalClientsAccess(kubeClusterClient, kcpClusterClient, bootstrapKcpClusterClient),
```

Comment:
I don't understand when we need a bootstrap client and when we don't.

Reply:
let me see if i can undo all the bootstrap client bits and move them into the branch where i'll properly set up multi-shard, standalone vw.